src/plugins/mer/meremulatordevicedconfoptionswidget.h

```diff
@@ -0,0 +1,70 @@
+/****************************************************************************
+**
+** Copyright (C) 2012 - 2018 Jolla Ltd.
+** Contact: http://jolla.com/
+**
+** This file is part of Qt Creator.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** Other Usage
+**
+** Alternatively, this file may be used in accordance with the terms and
+** conditions contained in a signed written agreement between you and Digia.
+**
+****************************************************************************/
+
+#ifndef MEREMULATORDEVICEDCONFOPTIONSWIDGET_H
+#define MEREMULATORDEVICEDCONFOPTIONSWIDGET_H
+
+#include <QTabWidget>
+
+namespace Mer {
+namespace Internal {
+
+namespace Ui {
+class MerEmulatorDeviceDconfOptionsWidget;
+}
+
+class MerEmulatorDeviceDconfOptionsWidget : public QTabWidget
+{
+    Q_OBJECT
+
+    Q_PROPERTY(bool readOnly READ isReadOnly WRITE setReadOnly)
+    Q_PROPERTY(QString options READ options WRITE setOptions NOTIFY optionsChanged)
```

Comment:
Do you still use these somewhere? If not, I would omit them.

Reply:
Removed.